src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -1591,19 +1016,27 @@ std::unique_ptr<LLMPipelineImplBase> LLMPipelineFactory::create(const std::share
     const auto pipeline_mode = str_to_pipeline(pop_or_default(properties_copy, "STATIC_PIPELINE", std::string("STATEFUL")));
     if (pipeline_mode == StaticPipelineKind::STATEFUL) {
         return std::make_unique<ov::genai::static_llm::StatefulLLMPipeline>(model,
-                                                                            model_desc,
                                                                             tokenizer,
                                                                             device,
                                                                             properties_copy,
                                                                             generation_config);
     }
-    return std::make_unique<ov::genai::static_llm::StatelessLLMPipeline>(model,
-                                                                         model_desc,
-                                                                         tokenizer,
-                                                                         device,
-                                                                         properties_copy,
-                                                                         generation_config);
+    OPENVINO_ASSERT(false);
+}
+
+std::unique_ptr<LLMPipelineImplBase>
+LLMPipelineFactory::create(const std::filesystem::path& models_path,
+                           const ov::genai::Tokenizer& tokenizer,
+                           const std::string& device,
+                           const ov::AnyMap& config) {
+    auto properties = config;
+    const auto pipeline_mode = str_to_pipeline(pop_or_default(properties, "STATIC_PIPELINE", std::string("STATEFUL")));
+    if (pipeline_mode == StaticPipelineKind::STATEFUL) {
```

Comment:
Same question

Reply:
Well, it depends if we want to extend it in future or not

I'd leave so far, as this change won't affect anything